.travis.yml

```diff
@@ -3,6 +3,15 @@ python:
   - 3.6
 notifications:
   email: false
+env:
+  global:
+    - USER="fred3m"
+    - EMAIL="fred.moolekamp@gmail.com"
+    - GH_REPO="github.com/fred3m/scarlet_extensions"
```

Comment:
I'd prefer if we called this variable differently. it might confuse our future selves because this *isn't* the scarlet repo